### PR TITLE
fix: unitless zero for `<angle>` values are allowed in CSS gradients

### PR DESCRIPTION
**Pull request by yisibl**
Problems found during lightningcss discussion: https://github.com/parcel-bundler/lightningcss/issues/298

### CODE REVIEW

File: test/cases/gradient.out.css [hunk 1]
```diff
@@ -220,3 +220,37 @@ div {
...
+  background: -o-repeating-linear-gradient(bottom, blue, red 33.3%);
+  background: repeating-linear-gradient(0, blue, red 33.3%)
+}
+
+.zero-grad {
+  background: -webkit-gradient(linear, left bottom, left top, from(green), to(blue));
+  background: -webkit-linear-gradient(bottom, green, blue);
+  background: -o-linear-gradient(bottom, green, blue);
+  background: linear-gradient(0grad, green, blue);
+  background-image: -webkit-repeating-linear-gradient(bottom, blue, red 33.3%);
```
Comment (yisibl) [inline test/cases/gradient.out.css:239]: I don't know why the test failed here, the single test(`test.only('changes angle in gradient'`) was passed.

Comment (ai) [reply test/cases/gradient.out.css:239]: It was prefixes cleaning test which failed.

Seems like we forgot to think about cleaning repeating gradients when we added their support.

Here is my fix 542bd85

**Comment by ai** [comment]
Thanks. Released in 10.4.12.

File: test/cases/gradient.out.css (hunk 1 above)
Comment (yisibl) [reply test/cases/gradient.out.css:239]: Nice

**Pull request merged**